scrapy/extensions/feedexport.py

```diff
@@ -480,6 +480,8 @@ def _handle_store_error(self, f, logmsg, spider, slot_type):
 
     def _handle_store_success(self, f, logmsg, spider, slot_type):
         logger.info("Stored %s", logmsg, extra={"spider": spider})
+        feed_uri = logmsg.split(":")[-1].strip()
```

Comment:
Avoid this type of text transformation, is quite possible it will break on the future if the format changes. If you need `slot.uri` you can pass it as an additional argument